Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -46,6 +48,7 @@ public NodejsProjectNode(NodejsProjectPackage package) : base(package, null)
             InitNodejsProjectImages();
 #pragma warning restore 0612
 
+            infoBarManager = new MissingNodeInfoBar(package);
```

Comment:
>package [](start = 52, length = 7)

Does the base type expose `package`?  If so, would it be preferable to create the `MissingNodeInfoBar` on demand?